tests/util/util_list_parts.sh

```diff
@@ -118,11 +118,18 @@ upload_check_parts() {
     log 2 "error checking part list before part upload"
     return 1
   fi
+  sleep 5
```

Comment:
Can you please put a comment in for this so we know what you are waiting for?  Makes it easier to diagnose weird timing failures later.